packages/flutter/test/material/about_test.dart

```diff
@@ -225,7 +256,12 @@ void main() {
 
     await tester.pumpAndSettle();
 
-    expect(tester.getTopLeft(find.text('DEF')), const Offset(8.0 + safeareaPadding, 287.0));
+    // The position of the top left of app bar title should indicate whether
+    // the safe area is sufficiently respected.
+    expect(
+      tester.getTopLeft(find.text('Licenses')),
+      const Offset(16.0 + safeareaPadding, 18 + safeareaPadding),
```

Comment:
Be consistent about doubles: either use `16`, or `18.0`.

Reply:
Okay, I'll go through the change list and conform to `18.0` everywhere. On closer inspection it seems to be the convention.